app/code/Magento/Catalog/Ui/Component/Listing/Columns/Websites.php

```diff
@@ -20,29 +21,42 @@ class Websites extends \Magento\Ui\Component\Listing\Columns\Column
      */
     const NAME = 'websites';
 
+    /**
+     * Data for concatenated website names value
+     */
+    const WEBSITE_NAMES = 'website_names';
+
     /**
      * Store manager
      *
      * @var StoreManagerInterface
      */
     protected $storeManager;
 
+    /**
+     * @var \Magento\Framework\DB\Helper
+     */
+    protected $_resourceHelper;
+
     /**
      * @param ContextInterface $context
      * @param UiComponentFactory $uiComponentFactory
      * @param StoreManagerInterface $storeManager
+     * @param Helper $resourceHelper
      * @param array $components
      * @param array $data
      */
     public function __construct(
         ContextInterface $context,
         UiComponentFactory $uiComponentFactory,
         StoreManagerInterface $storeManager,
+        Helper $resourceHelper,
```

Comment:
Please, do not change arguments order. This change goes against backward compatibility. Add the resourceHelper to the very end of the arguments set